PIL/PngImagePlugin.py

```diff
@@ -687,6 +687,9 @@ def _save(im, fp, filename, chunk=putchunk, check=0):
             mode = "%s;%d" % (mode, bits)
 
     # encoder options
+    if (im.info.get('showing', False)):
+        im.encoderinfo['compress_level'] = 1
+        im.info.pop('showing', None)
```

Comment:
`**options` turn into `encoderinfo` in the save method, so you can call save like: `im.save('foo.png', compress_level=1)` to pass the compression level into the PNG encoder.